README.md

```diff
@@ -1,6 +1,4 @@
-[TOC]
-
-# Introduction {#Introduction}
+# Brion
```

Comment:
Have you verified how does this look in Doxygen? I think that with your change the project name appears twice.